README.md

```diff
@@ -21,7 +21,25 @@ go build github.com/giantswarm/azure-operator
 
 ## Running azure-operator
 
-See [this guide][examples-local].
+Create an azure-operator role (replace SUBSCRIPTION_ID):
+
+```bash
+az role definition create --role-definition '{"Name":"azure-operator","Description":"Role for github.com/giantswarm/azure-operator","Actions":["*"],"NotActions":["Microsoft.Authorization/elevateAccess/Action"],"AssignableScopes":["/subscriptions/${SUBSCRIPTION_ID}"]}'
+```
+
+If you have a service proivder you want to reuse add the azure-operator role:
+
+```bash
+az role assignment create --assignee 4f4e6e7b-9d1f-4754-8b86-03cd7a59d873 --role azure-operator
```

Comment:
`--assignee ${SERVICE_PRINCIPAL_ID}` ?

Reply:
I renamed it to CLIENT_ID. It's the name we use everywhere else: https://github.com/giantswarm/azure-operator/blob/80c5c34f0cc0c06be61f77abb63b8440254779fe/main.go#L146